lib/slack/lookups.ex

```diff
@@ -1,57 +1,43 @@
 defmodule Slack.Lookups do
+  
+  alias Slack.Client
+  
   @doc ~S"""
   Turns a string like `"@USER_NAME"` into the ID that Slack understands (`"U…"`).
   """
-  def lookup_user_id("@" <> user_name, slack) do
```

Comment:
Curious on your thoughts on this, what do you think of keeping the lookup logic in here instead of in `client` to help keep `client` smaller/more focused?